src/views/GovernanceReviewTeam/Summary/index.tsx

```diff
@@ -18,7 +19,13 @@ import {
   translateRequestType
 } from 'utils/systemIntake';
 
-const RequestSummary = ({ intake }: { intake: SystemIntakeForm }) => {
+const RequestSummary = ({
+  intake,
+  oldIntake
+}: {
+  intake: GetSystemIntake;
+  oldIntake: SystemIntakeForm;
```

Comment:
Adding oldIntake here until we can migrate the admin lead saga to use a graph mutation instead. It wasn't quite working when I passed it the graph version of the intake, and rather than fight it this is a bandaid until we can do the next bit of work (which should be right after this gets merged).